app/models/physiqual/token.rb

```diff
@@ -6,6 +6,7 @@ class Token < ActiveRecord::Base
     # validates :token, presence: true
     # validates :refresh_token, presence: true
     validates :physiqual_user_id, presence: true
+    validates_uniqueness_of :physiqual_user_id, scope: :type
```

Comment:
Willen we dit ook in de database zelf constrainen dmv een unique index op [:physiqual_user_id, :type]?